sdk/storage/Microsoft.Azure.WebJobs.Extensions.Storage.Common/src/Shared/Queues/QueueProcessorOptions.cs

```diff
@@ -28,7 +28,7 @@ internal QueueProcessorOptions(QueueClient queue, ILoggerFactory loggerFactory,
         {
             Queue = queue ?? throw new ArgumentNullException(nameof(queue));
             PoisonQueue = poisonQueue;
-            Logger = loggerFactory?.CreateLogger(LogCategories.CreateTriggerCategory("Queue"));
+            Logger = loggerFactory?.CreateLogger<QueueProcessor>();
```

Comment:
Intentionally used QueueProcessor rather than QueueProcessorOptions as that fits better with the usage.